documentation/asciidoc/computers/camera/csi-2-usage.adoc

```diff
@@ -1,18 +1,18 @@
-== Camera serial interface 2 (CSI2): Unicam
+== Unicam
 
-The SoCs used on the Raspberry Pi range all have two camera interfaces that support either CSI-2 D-PHY 1.1 or Compact Camera Port 2 (CCP2) sources. This interface is known by the codename Unicam. The first instance of Unicam supports two CSI-2 data lanes, while the second supports four. Each lane can run at up to 1Gbit/s (DDR, so the max link frequency is 500MHz).
+Raspberry Pi SoCs all have two camera interfaces that support either CSI-2 D-PHY 1.1 or Compact Camera Port 2 (CCP2) sources. This interface is known by the codename Unicam. The first instance of Unicam supports two CSI-2 data lanes, while the second supports four. Each lane can run at up to 1Gbit/s (DDR, so the max link frequency is 500MHz).
```

Comment:
I guess this is applicable to pre-Pi-5; whereas on Pi5 we have two four-lane CSI-2 interfaces? https://www.raspberrypi.com/products/raspberry-pi-5/

Reply:
(But I don't know if Pi5 still counts as "Unicam" or not)

Reply:
Excellent point, I'll reach out internally. That term could probably use some work, if true...